server/controllers/finance/reports/operating/index.js

```diff
@@ -48,9 +49,8 @@ function getQueryExpense() {
   `;
 }
 
-function document(req, res, next) {
+async function document(req, res, next) {
```

Comment:
It seems like this `async` is out of place.  Normally, you would use `await` with `async`, but it doesn't look like you've gone that far.  Could you remove it?